springboard/templates/base.jinja2

```diff
@@ -14,17 +14,16 @@
     <div id="header">
         <div class="lang">
             <div>
-                {% for code, name in languages %}
-                <span>
-                    <a href="/locale/?language={{code}}">{{name}}</a>
-                    {% if not forloop.last %}
-                        <span> |</span>
-                    {% endif %}
-                </span>
+                {% for lang_code in view.get_display_languages() %}
```

Comment:
Hmmm. The problem I was trying to fix was that the ordering of the languages is not defined, so the selected language is not displayed first. But my change seems to break many things /o\. I think I'm going to see if I can sort this out in the template